template-only-bin/install-template.sh

```diff
@@ -11,4 +13,8 @@ cp -r \
   template-infra/Makefile \
   .
 
+echo "Remove files relevant only to template development"
 rm .github/workflows/template-only-*
+
+echo "Clean up template-infra folder"
+rm -fr template-infra
```

Comment:
👀

Reply:
just needed to re-merge from main, looks like i may have branched off a feature branch rather than from main